packages/form-field/src/FormField/FormField.styles.ts

```diff
@@ -30,9 +30,12 @@ export const getFontSize = ({
   }
 };
 
-export const labelTextContainerStyle = css`
+export const textContainerStyle = css`
```

Comment:
non-blocking system-related question: have we put any thought into having layout components like a `VStack` in this case?